openid.html

```diff
@@ -126,15 +135,17 @@
         var discovery = $("#node-config-input-discovery").val();
         var clientId = $("#node-config-input-clientId").val();
         var clientSecret = $("#node-config-input-clientSecret").val();
-        var url = 'openid-credentials/auth?id='+id+'&discovery='+encodeURIComponent(discovery)+'&clientId='+clientId+"&clientSecret="+clientSecret+"&callback="+encodeURIComponent(callback);
+        var scopes = $('#node-config-input-scopes').val();
+        var url = 'openid-credentials/auth?id='+id+'&discovery='+encodeURIComponent(discovery)+'&clientId='+clientId+"&clientSecret="+clientSecret+"&scopes="+scopes+"&callback="+encodeURIComponent(callback);
         $(this).attr("href",url);
         window.openidConfigNodeIntervalId = window.setTimeout(pollOpenIDCredentials, 2000);
       });
       $("#node-config-start-auth").click(function(e) {
         var discovery = $("#node-config-input-discovery").val();
         var clientId = $("#node-config-input-clientId").val();
         var clientSecret = $("#node-config-input-clientSecret").val();
-        if (discovery === "" || clientId === "" || clientSecret === "") {
+        var scopes = $('#node-config-input-scopes').val();
+        if (discovery === "" || clientId === "" || clientSecret === "" || scopes === "") {
```

Comment:
Scope should not be mandatory:

```suggestion
        if (discovery === "" || clientId === "" || clientSecret === "") {
```